### PR TITLE
[minor] Improve plotting documentation

### PR DESCRIPTION
**Pull request by xmatthias**
## Summary
Slightly improve the plotting documentation (add a missing - but important sample), standardize how to call the scripts (`python3 script/<scripname.py`).

### CODE REVIEW

File: docs/plotting.md [hunk 1]
```diff
@@ -1,63 +1,79 @@
...
 Timerange doesn't work with live data.
 
 To plot trades stored in a database use `--db-url` argument:
+
+``` bash
+python3 scripts/plot_dataframe.py --db-url sqlite:///tradesv3.dry_run.sqlite -p BTC/ETH
 ```
-python scripts/plot_dataframe.py --db-url sqlite:///tradesv3.dry_run.sqlite -p BTC/ETH
+
+To polt trades from a backtesting result, use `--export-filename <filename>`
```
Comment (hroff-1902) [inline docs/plotting.md]: To plot

File: docs/plotting.md [hunk 2]
```diff
@@ -1,63 +1,79 @@
...
+
+``` bash
+python3 scripts/plot_dataframe.py --export-filename user_data/backtest_data/backtest-result.json -p BTC/ETH
 ```
 
 To plot a test strategy the strategy should have first be backtested.
```
Comment (hroff-1902) [inline docs/plotting.md]: To plot a custom strategy...

File: docs/plotting.md [hunk 3]
```diff
@@ -1,63 +1,79 @@
...
-```
-python scripts/plot_dataframe.py -s Strategy_Name -p BTC/ETH --datadir user_data/data/<exchange_name>/
+
+``` bash
+python3 scripts/plot_dataframe.py -s Strategy_Name -p BTC/ETH --datadir user_data/data/<exchange_name>/
 ```
 
 ## Plot profit
 
 The profit plotter show a picture with three plots:
```
Comment (hroff-1902) [inline docs/plotting.md]: shows

File: docs/plotting.md [hunk 4]
```diff
@@ -1,63 +1,79 @@
 # Plotting
+
 This page explains how to plot prices, indicator, profits.
```
Comment (hroff-1902) [inline docs/plotting.md]: indicators and profits.

File: docs/plotting.md [hunk 5]
```diff
@@ -1,63 +1,79 @@
...
 
 Plotting scripts use Plotly library. Install/upgrade it with:
 
-```
-pip install --upgrade plotly
+``` bash
+pip install -U -r requirements-plot.txt
 ```
 
 At least version 2.3.0 is required.
```
Comment (hroff-1902) [inline docs/plotting.md]: Is not this sentence redundant now and should be removed if requirements-plot.txt is now used? version 2.3.0 of what package, if reference to plotly is removed?

File: docs/plotting.md [hunk 6]
```diff
@@ -1,63 +1,79 @@
...
 
 Example
-```
-python scripts/plot_dataframe.py -p BTC/ETH
+
+``` bash
+python3 scripts/plot_dataframe.py -p BTC/ETH
 ```
 
 The `-p` pairs argument, can be used to specify
```
Comment (hroff-1902) [inline docs/plotting.md]: remove comma

File: docs/plotting.md [hunk 7]
```diff
@@ -1,63 +1,79 @@
...
-pip install --upgrade plotly
+``` bash
+pip install -U -r requirements-plot.txt
 ```
 
 At least version 2.3.0 is required.
 
 ## Plot price and indicators
+
 Usage for the price plotter:
```
Comment (hroff-1902) [inline docs/plotting.md]: And how to plot indicators as the section says? Did not find this below...

File: docs/plotting.md [hunk 8]
```diff
@@ -76,13 +92,14 @@ that makes profit spikes.
```
Comment (hroff-1902) [inline docs/plotting.md]: ^ your algorithm

File: docs/plotting.md (hunk 7 above)
Comment (xmatthias) [reply docs/plotting.md]: mmmh good point :+1:

File: docs/plotting.md (hunk 8 above)
Comment (xmatthias) [reply docs/plotting.md]: sorry i don't understand this?

Comment (hroff-1902) [reply docs/plotting.md]: It's written In the sentence above this line "show how you algorithm works". Change "you algorithm" --> "your algorithm" there. Cannot press a "+" button straight on that line so made this comment below.

**Comment by hroff-1902** [comment]
LGTM 👍

**Pull request merged**